ansible/roles/test/tasks/pfc_wd.yml

```diff
@@ -168,6 +168,9 @@
       when: warm_reboot_test | bool == false
 
     - block:
+        - name: Set to default PFC WD config for warm-reboot test
+          shell: bash -c 'sudo pfcwd stop; sleep 5; sudo pfcwd start_default'
```

Comment:
Just noticed that https://github.com/Azure/sonic-utilities/blob/master/pfcwd/main.py#L231 gets default_pfcwd_status

In public image and minigraph generated by testbed-cli.sh this is disable
```
admin@arc-switch1004:~$ redis-cli -n 4 hget 'DEVICE_METADATA|localhost' 'default_pfcwd_status'
"disable"
```
So it will not start pfcwd